### DISCUSSION TITLE
Ignore Folder

### DISCUSSION THREAD

**Discussion by Wolface3** [Difegue/LANraragi#1120] [Q&A] [answered]
Is there a way to ignore a folder/ directory exclusion within my content folder? I sometimes have different copies of one archive because of translation groups, quality differences and such but I don't want them to be displayed in Lanraragi. Is there something similar to Plex's ".plexignore" file I can use?

**Comment by Difegue** [comment, marked as answer]
That's not a feature at the moment, but I don't think it would be too hard for anyone to implement.

**Comment by groutoutlook** [comment]
Hi, have we do anything relate to this yet?
